wandb/sdk/lib/apikey.py

```diff
@@ -32,8 +35,16 @@
     LOGIN_CHOICE_DRYRUN,
 ]
 
+
+class _NetrcPermissions(Enum):
+    NETRC_EXISTS = os.F_OK
+    NETRC_READ_ACCESS = os.R_OK
+    NETRC_WRITE_ACCESS = os.W_OK
```

Comment:
If this is always used in a dict like

```python
{
    _NetrcPermissions.NETRC_EXISTS: ...,
    _NetrcPermissions.NETRC_READ_ACCESS: ...,
    _NetrcPermissions.NETRC_WRITE_ACCESS: ...,
}
```

then these are not enums, but fields on a dataclass.

```python
@dataclasses.dataclass(frozen=True)
class _NetrcPermissions:
    exists: bool
    readable: bool
    writeable: bool
```

Otherwise, you can't write `netrc_access[_NetrcPermissions.NETRC_WRITE_ACCESS]` without handling the `KeyError` (though that seems meaningless)